ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -10,18 +10,64 @@ include 'pan/types';
 
 type ssh_preferred_authentication = string with match(SELF, '^(gssapi-with-mic|hostbased|publickey|keyboard-interactive|password)$');
 
+
+type ssh_ciphers = string with is_valid_ssh_cipher(SELF);
```

Comment:
you introduce a new type that already exists in `metacofnig/ssh` and it is not used in this template. make an issue to resolve this, and add a comment with the issue number here, and comment this type for now.
you can already check that the current definition in metaconfig /ssh is the same regexp

Reply:
what exactly do I need to resolve?

This way of working was recommended to me by @wdpypere  To keep it backwards compatible.

Reply:
I did what now?

Reply:
the types are unique. haveing this type here implies that you cannot include the metaconfig service that has the same type defined. there can be only one...